moped-editor/src/auth/user.js

```diff
@@ -109,7 +109,6 @@ export const initializeUserDBObject = (userObject) => {
     }).then((res) => {
       // Then we parse the response
       res.json().then((resData) => {
-        // If we have any user data,  use it
```

Comment:
i guess [this patch commit](https://github.com/cityofaustin/atd-moped/pull/1076/commits/9a312d00c38d242a6b5c25e4102e26ac63f8aa1b) never made it's way back to main